styles/components/avatar.css

```diff
@@ -0,0 +1,40 @@
+.avatar {
+  position: relative;
+  margin: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.avatar__img--rounded {
+  object-fit: cover;
+  border-radius: 50%;
+}
+.avatar__img--squared {
+  object-fit: cover;
+  border-radius: 12.5%;
+}
+.avatar__badge {
+  position: absolute;
+  background: green;
+  border-radius: 25px;
```

Comment:
Try to use same units throughout.

Reply:
Changed that to use relative percentage values.